### PR TITLE
Adding Deployment Instructions for Patterns

### LINKED ISSUE #8: [Docs] Add deployment instructions for patterns
**Current State:**
The readme does not have any information about the deployment of patterns

**Desired State:**
Add a section about(Actions in Patterns UI) deployment for patterns. 

**Reference Docs**
https://docs.meshery.io/functionality/pattern-management

---
**Contributor Resources**
- [Meshery documentation site](https://docs.meshery.io/)
- [Meshery documentation source](https://github.com/layer5io/meshery/tree/master/docs)
- [Instructions for contributing to documentation](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)

### PR DESCRIPTION
**Pull request by Deepansharora27**
**Description**

This PR fixes #8 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Deepansharora27** [comment]
@warunicorn19 @Chadha93 Done with the Issue #8

**Comment by Deepansharora27** [comment]
Also tagging @leecalcote for the update

**Review by Chadha93 — CHANGES_REQUESTED** [review]
@Deepansharora27 By instructions we meant to just add the basic information about it in a gist, you don't have to copy all the information from the docs just referencing it with URLs will work.

@leecalcote @navendu-pottekkat would you please throw some light on this?

**Comment by Deepansharora27** [comment]
@Chadha93 Hi Gaurav, Kindly tell me then what part of information should I take into account for embedding into README file?

**Comment by navendu-pottekkat** [comment]
Instructions on how to import pattern files that are stored in this repo through the UI and mesheryctl would be enough here.

You can point to the pattern files in this repo for examples.

After outlining the instructions briefly, you can point to the the Meshery docs.

**Comment by leecalcote** [comment]
Thanks for jumping in here, @Deepansharora27 👍 

Will you remove the `.DS_Store` files and add an entry in the .gitignore file for these?

**Comment by Deepansharora27** [comment]
@leecalcote Roger that !!
Done with it

**Comment by Deepansharora27** [comment]
@navendu-pottekkat @Chadha93 I am done with the Changes,Can you please have a look ?

**Review by Chadha93 — APPROVED** [review]
@Deepansharora27 Cool Stuff :1st_place_medal:

**Review by l5io — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -13,3 +14,4 @@
 
 # Dependency directories (remove the comment below to include it)
 # vendor/
+.DS_Store
```
Comment (leecalcote) [inline .gitignore:17]: Nice on, @Deepansharora27

**Review by leecalcote — APPROVED** [review]
Good work, @Deepansharora27

**Pull request merged**